reporting/transformer_example/src/conftest.py

```diff
@@ -0,0 +1,7 @@
+import pytest
+from elasticsearch import Elasticsearch
+
+
+@pytest.fixture
+def elasticsearch_client(elasticsearch_url):
```

Comment:
Suggestion: putting this in `shared_conftest.py` would make it usable in other applications.